storage/s3-sdk/s3_sdk_test.py

```diff
@@ -70,7 +70,8 @@ def test_blob(test_bucket):
 @backoff.on_exception(backoff.constant, ClientError, interval=1, max_time=15)
 def test_list_buckets(capsys, hmac_fixture, test_bucket):
     list_gcs_buckets.list_gcs_buckets(
-        google_access_key_id=hmac_fixture[0].access_id, google_access_key_secret=hmac_fixture[1]
+        google_access_key_id=hmac_fixture[0].access_id,
```

Comment:
Revert this change unless the lint required it?

Reply:
Yep running lint made this change